src/components/project-change-request/project-change-request.repository.ts

```diff
@@ -60,18 +64,20 @@ export class ProjectChangeRequestRepository extends DtoRepository(
         node('project', 'Project'),
       ])
       .apply(matchPropsAndProjectSensAndScopedRoles(session))
-      .return<{ dto: UnsecuredDto<ProjectChangeRequest> }>(
-        merge('props', {
-          scope: 'scopedRoles',
-          canEdit: `props.status = "${Status.Pending}"`,
-        }).as('dto')
+      .return<{ dto: UnsecuredDto<ProjectChangeRequest>; scope: ScopedRole[] }>(
+        [
+          merge('props', {
+            canEdit: `props.status = "${Status.Pending}"`,
+          }).as('dto'),
+          'scopedRoles as scope',
+        ]
       );
     const result = await query.first();
     if (!result) {
       throw new NotFoundException('Could not find project change request');
     }
 
-    return result.dto;
+    return { ...result.dto, scope: result.scope };
```

Comment:
This change doesn't do anything...

Reply:
yeah, fixed that already and going to push after testing